torch/nn/modules/rnn.py

```diff
@@ -506,8 +506,8 @@ class GRUCell(RNNCellBase):
     .. math::
 
         \begin{array}{ll}
-        r = sigmoid(W_{ir} x + b_{ir} + W_{hr} h + b_{hr}) \\
-        i = sigmoid(W_{ii} x + b_{ii} + W_{hi} h + b_{hi}) \\
+        r = \mathrm{sigmoid}(W_{ir} x + b_{ir} + W_{hr} h + b_{hr}) \\
+        i = \mathrm{sigmoid}(W_{ii} x + b_{ii} + W_{hi} h + b_{hi}) \\
```

Comment:
I forgot to fix the GRUcell's. I'll fix if it needs.

 I just found `z` notation in a Japanese book on NLP so I'm not sure if it is good way. But I think it's clearer than repetition of `i`s in different meaning.